common/views/components/Buttons/Control/Control.js

```diff
@@ -70,10 +76,25 @@ const Control = forwardRef(
     }
 
     return (
-      // $FlowFixMe (conditional JSX tag)
-      <HtmlTag ref={ref} {...attrs}>
-        <InnerControl text={text} icon={icon} />
-      </HtmlTag>
+      <>
+        {nextLink ? (
+          <NextLink
+            {...nextLink}
+            scroll={scroll}
+            replace={replace}
+            prefetch={prefetch}
+          >
+            <a ref={ref} {...attrs}>
+              <InnerControl text={text} icon={icon} />
+            </a>
+          </NextLink>
+        ) : (
+          // $FlowFixMe (conditional element)
```

Comment:
What's this?

Reply:
I'm not sure (it was pre-existing rage-`$flowFixMe`). It didn't make sense to me, but it's possible I'd overlooked something.